packages/web/src/components/card/Card.tsx

```diff
@@ -0,0 +1,56 @@
+import { ReactNode, Ref, forwardRef } from 'react'
+
+import { Divider, Flex, Paper, PaperProps } from '@audius/harmony'
+
+type CardSize = 's' | 'm' | 'l'
+
+const cardSizes = {
+  s: 200,
+  m: 224,
+  l: 320
+}
+
+export type CardProps = PaperProps & {
+  size: CardSize
+}
+
+export const Card = forwardRef((props: CardProps, ref: Ref<HTMLDivElement>) => {
+  const { size, children, ...other } = props
+  return (
+    <Paper
+      ref={ref}
+      role='button'
+      tabIndex={0}
+      direction='column'
+      border='default'
+      w={cardSizes[size]}
```

Comment:
is height meant to be standard or dependent on the content?

Reply:
dependent on content interestingly